python/tk_framework_desktopserver/shotgun/scripts/execute_command.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import sys
+import cPickle
+import os
+
+def execute(config, project, name, entities, base_configuration, engine_name):
+    import sgtk
+
```

Comment:
Could we consider having the logs be sent directly to the main desktop process instead of a file? We would have to provide the RPC credentials. I'm worried too many writers to the file is going to cause an issue at some point. Especially on Windows. OR set up an endpoint on the websocket server for logging. (I like that a lot more actually!)